cpp/tests/strings/fixed_point_tests.cpp

```diff
@@ -218,13 +219,9 @@ TEST_F(StringsConvertTest, IsFixedPoint)
   CUDF_TEST_EXPECT_COLUMNS_EQUIVALENT(*results, expected64_scaled);
 }
 
-#include <cudf/strings/convert/convert_floats.hpp>
-
 TEST_F(StringsConvertTest, FloatFixedPoint)
```

Comment:
Actually, this whole test-clause was just from some earlier debugging I did and was commited by mistake. You will notice that it does not actually check any results. Perhaps you could remove this and the include as well? Otherwise, I can remove it in a separate PR.

Reply:
Done.